src/components/core/tabs/stories/tabs.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Story } from '@storybook/react';
+import type { TabsProps } from '../tabs';
+import Tabs from '../tabs';
+import Tab from '../tab';
+import DepositIconLight from '../../../../images/svg/modal/ic-deposit-light.svg';
+import WithdrawIconLight from '../../../../images/svg/modal/ic-withdraw-light.svg';
+import TransferIconLight from '../../../../images/svg/modal/ic-transfer-light.svg';
+import TransactionIconLight from '../../../../images/svg/modal/ic-transactions-light.svg';
+import DepositIconDark from '../../../../images/svg/modal/ic-deposit-dark.svg';
+import WithdrawIconDark from '../../../../images/svg/modal/ic-withdraw-dark.svg';
+import TransferIconDark from '../../../../images/svg/modal/ic-transfer-dark.svg';
+import TransactionIconDark from '../../../../images/svg/modal/ic-transactions-dark.svg';
```

Comment:
Let's add the webpack config for aliases for absolute import so that we can avoid having to do this. @balakrishna-binary @yashim-deriv

Reply:
Yes agree to this @sean-binary. I will add the aliases @balakrishna-binary @yashim-deriv.

Reply:
Thanks @farrah-deriv. When you setup alias, please include this library - https://www.npmjs.com/package/tsconfig-paths-webpack-plugin, so that we just can maintain paths in one place(tsconfig file).